samples/buildrun/buildrun_buildkit_cr.yaml

```diff
@@ -1,10 +1,9 @@
 ---
-apiVersion: shipwright.io/v1alpha1
+apiVersion: shipwright.io/v1beta1
 kind: BuildRun
 metadata:
   name: buildkit-buildrun
 spec:
-  buildRef:
+  build:
     name: buildkit-build
-  serviceAccount:
-    generate: true
+  serviceAccount: ""
```

Comment:
@qu1queee @SaschaSchwarze0  Thank you for helping out with my queries on Slack. I understand our perspective w.r.t `serviceAccount.generate` atm. I would still need your help here to determine what value should be the best for the `serviceAccount` as per the [v1beta1 fields](https://github.com/shipwright-io/build/blob/main/pkg/apis/build/v1beta1/buildrun_types.go#L47).

What should name we expect for he serviceAccounts, or, it's better to leave it blank and let it pick up the default SA?

Reply:
See https://github.com/shipwright-io/build/pull/1352#discussion_r1286906633

Reply:
Thanks!